lib/home.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:movie_db_app/widgets/description.dart';
+import 'package:movie_db_app/widgets/header.dart';
+import 'package:movie_db_app/widgets/subheader.dart';
+import '../shared/constants.dart';
+
+class Home extends StatefulWidget {
+  const Home({
+    super.key,
+  });
+
+  @override
+  State<Home> createState() => _HomeState();
+}
+
+class _HomeState extends State<Home> {
+  int _likeCounter = 0;
+  static const String appName = 'The MovieDB App';
+  static const String movieOverview = 'While working underground to fix a water main, Brooklyn plumbers—and brothers—Mario and Luigi are transported down a mysterious pipe and wander into a magical new world. But when the brothers are separated, Mario embarks on an epic quest to find Luigi.';
+  static const String voteAverage = '7.8';
+  static const List<String> genres = ['Adventure', 'Animation', 'Comedy', 'Family', 'Fantasy',];
+  static const String originalTitle = 'Original Title: The Super Mario Bros. Movie';
+  static const String dateRelease = '2023-04-05';
+  static const String movieTitle = 'The Super Mario Bros. Movie';
+  static const String imagePath = 'assets/images/poster.jpg';
+  static const String backdropPath = 'assets/images/backdrop.jpg';
```

Comment:
I would suggest to move all this static const to the movie constructor and create a factory method that return a Movie instance like:
```
Movie(
voteAverage: '7.8',
....
)
```
In the Home instead to use this constants you can use the movie object that you are going to create

Reply:
solved